config/routes.rb

```diff
@@ -171,6 +171,7 @@
   resources :lettings_logs, path: "/lettings-logs" do
     get "delete-confirmation", to: "lettings_logs#delete_confirmation"
     get "duplicate-logs", to: "duplicate_logs#show"
```

Comment:
gentle preference for these to be just duplicates but it's obv not very important